initramfs/init.sh

```diff
@@ -160,6 +160,12 @@ find_and_run_update()
             break
         fi
 
```

Comment:
Looks good, i think adding a debug tracing line would make it even better
echo "Moved the update image, unmount '${dev}' before performing the update."
And it should be unmount instead of umount.